.pre-commit-config.yaml

```diff
@@ -52,3 +52,7 @@ repos:
         files: ^(CHANGES.rst|development.rst|README.rst)$
         language: python
         additional_dependencies: [pygments, restructuredtext_lint]
+-   repo: git://github.com/elidupuis/mirrors-sass-lint
```

Comment:
:question:  actually i am not quite shre about this.
is tends to fail on my local machine when i call `tox -e linting`.

will do further investigation, soon.

Reply:
fixed